pkg/iac/supplier/supplier.go

```diff
@@ -28,7 +29,9 @@ func IsSupplierSupported(supplierKey string) bool {
 	return false
 }
 
-func GetIACSupplier(configs []config.SupplierConfig, library *terraform.ProviderLibrary, backendOpts *backend.Options) (resource.Supplier, error) {
+func GetIACSupplier(configs []config.SupplierConfig, library *terraform.ProviderLibrary, backendOpts *backend.Options, progress output.Progress) (resource.Supplier, error) {
+	progress.Start()
```

Comment:
shouldn't you stop in the same function ?

Reply:
I think it's the Resources() method that performs the state enumeration, so the progress stops there

Reply:
It's really confusing to know when the progess is started/stopper if it's not around the listing call.
Maybe it would be easier to start both bars in driftctl.go scan ?